src/components/WYSIWYG/WYSIWYG.scss

```diff
@@ -1,12 +1,13 @@
 @import '../../styles/settings/colors';
 
 .c-content table td {
-	border: 1px solid $color-gray-100;
+	border: 0.1rem solid $color-gray-100;
 }
 
 .c-rich-text-editor {
-	border: 1px solid #bac7d1;
+	border: 0.1rem solid $color-gray-150;
```

Comment:
shouldn't rems only be used for bigger values than 1px?
We don't want the border to become bigger if a user enables scaling in their browser, right?